app/core/components/LabeledTextField/index.tsx

```diff
@@ -29,7 +29,8 @@ export const LabeledTextField = ({
 }: LabeledTextFieldProps) => {
   const { error } = useField(name);
 
-  const [value, setValue] = useControlField<string>(name);
+  let [value, setValue] = useControlField<string>(name);
+  if (!value) value = "";
```

Comment:
I would assume that you don't need the `if (!value) value = "";` line. `useControlField` is supposed to assign the default value you send to the ValidatedForm. Why is it needed? If it is to fix the empty hours error on the members form, I found a better solution :-)

Reply:
Yes, its to fix that, but if there is a better solution, lets do that

Reply:
```diff
diff --git a/app/models/project.server.ts b/app/models/project.server.ts
index 2d41f16..ade4c90 100644
--- a/app/models/project.server.ts
+++ b/app/models/project.server.ts
@@ -235,7 +235,7 @@ export async function updateMembers(
   projectMembers: {
     id?: string;
     profileId: string;
-    hoursPerWeek?: number;
+    hoursPerWeek?: number | null;
     role?: { id: string }[];
     practicedSkills?: { id: string }[];
     active: boolean;
diff --git a/app/routes/projects/$projectId.members.tsx b/app/routes/projects/$projectId.members.tsx
index 8a70217..62dc397 100644
--- a/app/routes/projects/$projectId.members.tsx
+++ b/app/routes/projects/$projectId.members.tsx
@@ -61,13 +61,17 @@ export const validator = withZod(
             name: z.string().optional(),
           })
         ),
-        practicedSkills: z.array(
-          z.object({
-            id: z.string(),
-            name: z.string().optional(),
-          })
-        ),
-        hoursPerWeek: zfd.numeric(z.number().optional()),
+        practicedSkills: z
+          .array(
+            z.object({
+              id: z.string(),
+              name: z.string().optional(),
+            })
+          )
+          .optional(),
+        hoursPerWeek: zfd
+          .numeric(z.number().nullish())
+          .transform((v) => (v === undefined ? null : v)),
         active: zfd.checkbox(),
       })
     ),
```

Reply:
Fixed!